ERCS/erc-7208.md

```diff
@@ -271,6 +271,8 @@ We present an **educational example** implementation showcasing two types of tok
 **This example has not been audited and should not be used in production environments.**
 
 
+See [Minimalistic Data Index](https://github.com/NexeraProtocol/Minimalistic-Data-Index) implementation.
```

Comment:
The linter is correct here. For the reasons enumerated [here](https://ethereum.github.io/eipw/markdown-rel-links/), please remove this external link.

Reply:
Hi Sam,
I've removed the external link replacing it with the link to README.md, as you've suggested in the comment above.
But now the HTML prufer fails with
```
For the Links > Internal check, the following failures were found:
* At ./_site/EIPS/eip-7208.html:526:
  internally linking to ../assets/erc-7208/contracts/README.md, which does not exist
HTML-Proofer found 1 failure!
```
which was the reason we've tried with external repo.
Do you have any ideas how to fix it?